tests/Fixer/ClassNotation/VisibilityRequiredFixerTest.php

```diff
@@ -849,5 +849,13 @@ public function provideFix80Cases()
         yield [
             '<?php class Foo { private int | /* or empty */ null $foo; }',
         ];
+
+        yield [
+            '<?php class Foo { private array|null $foo; }',
+        ];
+
+        yield [
+            '<?php class Foo { private null|array $foo; }',
+        ];
```

Comment:
maybe also:

```suggestion
        yield [
            '<?php class Foo { private null|array $foo; }',
        ];

        yield [
            '<?php class Foo { public null|array $foo; }',
            '<?php class Foo { null|array $foo; }',
        ];
```

Reply:
`<?php class Foo { null|array $foo; }` is not valid syntax, I've added `static` keyword.